src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -3989,6 +3989,33 @@ class C
 {
     public void M()
     {
+        (((int, string))((1, ""hello""))).ToString();
+    }
+}";
+
+            await TestAsync(code, expected, index: 0, compareTokens: false, parseOptions: TestOptions.Regular.WithTuplesFeature(), withScriptOption: true);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)]
+        public async Task Tuples_02()
+        {
+            var code = @"
+using System;
+class C
+{
+    public void M()
+    {
+        var [||]x = (1, ""hello"");
+        x.ToString();
+    }
+}";
+
+            var expected = @"
+using System;
+class C
+{
+    public void M()
+    {
         ((1, ""hello"")).ToString();
```

Comment:
Question.  The parens shouldn't be required here, right?   If that's the case then @DustinCampbell, our simplifier should unparenthesize tuples.